services/collaboration/pkg/config/wopiapp.go

```diff
@@ -0,0 +1,7 @@
+package config
+
+// WopiApp defines the available configuration in order to connect to a WOPI app.
+type WopiApp struct {
+	Addr     string `yaml:"addr" env:"COLLABORATION_WOPIAPP_ADDR" desc:"The URL where the WOPI app is located, such as https://127.0.0.1:8080."`
+	Insecure bool   `yaml:"insecure" env:"COLLABORATION_WOPIAPP_INSECURE" desc:"Connect to the WOPI app insecurely."`
```

Comment:
Does this also refer to `OCIS_INSECURE`? I guess so. In case that should be changed to:
`OCIS_INSECURE;COLLABORATION_WOPIAPP_INSECURE`. The text should the be: `Whether to verify the server TLS certificates.`

Reply:
It's to skip certificate validation against the wopi app (onlyoffice, collabora, etc). I doubt it's related to the ocis configuration.
`Skip TLS certificate verification when connecting to the WOPI app` might be a better choice